themes/base/theme.css

```diff
@@ -119,7 +119,7 @@ a.ui-button:active,
 .ui-icon-background,
 .ui-state-active .ui-icon-background {
 	border: #003eff/*{borderColorActive}*/;
-	background-color: #ffffff/*{fcActive}*/;
+	background-color: #ffffff/*{bgColorContent}*/;
```

Comment:
Ok. I think it makes sense for these to match. Though it's a shame the active colors don't work here.